### PR TITLE
566 update case study

### LINKED ISSUE #566: update-case-study

### PR DESCRIPTION
**Pull request by Arfah**
#closes #566 

Please check and merge

### CODE REVIEW

File: data/alumni_videos.yml [hunk 1]
```diff
@@ -1,3 +1,6 @@
+- name: Maya Driver
+  occupation: Junior Developer at <a href="https://eu.deloittedigital.com/en/home">Deloitte Digital</a>
```
Comment (sjmog) [inline data/alumni_videos.yml]: Do you add `target="_blank"` to these links so they open in a new tab?

File: data/graduates.yml [hunk 2]
```diff
@@ -25,6 +25,120 @@
...
+#       answer:
+#     - question: How has your life changed since graduation?
+#       answer:
+#     - question: What advice would you give to people thinking of changing career?
+#       answer:
+#     - question: What do you say to people who say £8000 is expensive?
+#       answer:
+#     - question: Sum up your Makers Academy experience
+#       answer:
+
```
Comment (sjmog) [inline data/graduates.yml]: Why is this in here if it is commented?

Comment (nikeshashar) [inline data/graduates.yml]: @sjmog It's a template for us to use when making new additions to the YML file. Very very useful

Comment (sjmog) [inline data/graduates.yml]: No problemo. Just FYI - unless it's clear somehow it's a template, I would have deleted it as part of making a PR. Might be worth making that clearer somehow

File: data/alumni_videos.yml (hunk 1 above)
Comment (Arfah) [inline data/alumni_videos.yml]: yeah I guess so.. can you add it? I'm not quite sure where it goes.

**Comment by sjmog** [comment]
Done @Arfah - take a peek

File: data/graduates.yml (hunk 2 above)
Comment (nikeshashar) [inline data/graduates.yml]: Will add it in

**Comment by nikeshashar** [comment]
:shipit: (added the note about the template)

**Pull request merged**